tests/e2e/support/page/files/publicLink.ts

```diff
@@ -191,5 +193,9 @@ export class PublicLink {
     await this.publicLinkButtonLocator.click()
     await this.fillThePublicLinkForm({ name, password, role, dateOfExpiration })
     await this.createLinkButtonLocator.click()
+
+    lastCreatedPublicLink = await page.textContent(
+      `//ul/li//h5[contains(text(),'${name}')]/following-sibling::div/a`
+    )
```

Comment:
I think it would be good to implement setter and getter here.

Reply:
working on that